examples/features/sub-modules/inter-broker-bridge/README.md

```diff
@@ -0,0 +1,143 @@
+# Inter-broker Bridging
+
+An example project showing how to do different varieties of bridging with ActiveMQ brokers.
+
+## 5.x to Artemis Camel Bridge
+
+This is an example of using Camel in the 5.x broker to bridge messages to Artemis. There isn't anything to deploy in this
+example. It's just a set of instructions.
+
+### Prerequisites
+
+- install ActiveMQ 5.x
+- install ActiveMQ Artemis
+
+### Preparing
+
+1) Fetch the `artemis-jms-client-all` jar from [Maven Central](https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.apache.activemq%22%20AND%20a%3A%22artemis-jms-client-all%22)
```

Comment:
Should it not be directing people to Apache download as official place to get distribution

Reply:
If there was a place on the Apache website where one could download the artemis-jms-client-all jar that would be preferred, of course. I'm not aware of such a place since that jar is not in our distribution.

Reply:
It’s a good point we should then have a clients folder to have the client jars in as part of distro, we should sort that rather than pointing to here. I can submit something in next day or two to add a clients section in distro if no one beats me to it

Reply:
Including the "all" clients in the distro is a fine idea, but I don't see that as a blocker for this PR. I don't see a real problem with pointing people to Maven Central.

Reply:
I’m not saying it’s a blocker I’m saying I’d like to sort it before release